sessionctx/variable/session.go

```diff
@@ -202,7 +202,7 @@ type SessionVars struct {
 	// systems variables, don't modify it directly, use GetSystemVar/SetSystemVar method.
 	systems map[string]string
 	// PreparedStmts stores prepared statement.
-	PreparedStmts        map[uint32]*ast.Prepared
+	PreparedStmts        map[uint32]interface{}
```

Comment:
How about directly store `PrepareObject`?

Reply:
I've tried this. `PrepareObject` stores `visitInfo`(this in planner ) field, seems  `SessionVars` struct in `variable package` cannot  import planner related package, there would be cycle
@lysu  @zz-jason